Documentation/SUMMARY.md

```diff
@@ -0,0 +1,53 @@
+# Table of Contents
```

Comment:
consider renaming the `Documentation` directory to [`docs`](https://github.com/kriasoft/Folder-Structure-Conventions)